docs/fern/pages/reference/schemas/annotation-style.md

```
@@ -0,0 +1,47 @@
+
+## Referenced In
+
+- [annotation](/reference/schemas/annotation)
+- [Annotation fields](/reference/schemas/annotation-fields)
+- [Annotation defaults](/reference/schemas/annotation-defaults)
+
+## Fields
+
+Field | Type | Description | Default
+:-- | :-- | :-- | :--
+color | string | Optional. Foreground color — strokes, arrowheads, and text (hex, rgb, or named color). | 
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)